src/tests/Login.test.jsx

```diff
@@ -107,3 +107,25 @@ it('handles wrong credentials', async () => {
     
     expect(screen.getByLabelText('errorMessage')).toHaveTextContent("The server is currently unavailable. Try again later")
   })
+
+it('Checking if the fields are empty', async () => {
+
+    const{getByLabelText} = render(<Login />)
+
+    fireEvent.change(screen.getByLabelText("Username or Email:", {selector: "input"}), {target: { value: "" }})
+    
+    fireEvent.change(screen.getByLabelText("Password :", {selector: "input"}), {target: { value: "" }})
+    
+    act(() => {
```

Comment:
Please take your time when modifying the code base on the feedback given, @khushishikhu . You missed the one I pointed out to you [here](https://github.com/anitab-org/bridge-in-tech-web/pull/83#discussion_r481889755} about taking out `except` out of the `act` block. It's a good practice to double check the changes you make before you push again to make sure that you've addressed each point of the feedback.

Reply:
yeah pardon me. i didn't actually noticed that because my tests were passing. i will be more careful from next time